crates/artifact/src/docker.rs

```diff
@@ -172,7 +172,7 @@ pub struct DockerId(String);
 
 impl From<String> for DockerId {
     fn from(s: String) -> Self {
-        DockerId(s.trim().to_string())
+        DockerId(s.trim().split_whitespace().last().unwrap().to_string())
```

Comment:
👍 this is what I found was necessary to run on latest Docker (but not on my machine). This means we could likely add a CI task to do `benchmarks-next/build.sh` and then check if any benchmarks have changed to make sure the build is deterministic.

Reply:
Yeah, getting this in CI would be great.

Reply:
Let's do it as a part of #54